src/playercards/Tarotcard.ttslua

```diff
@@ -0,0 +1,19 @@
+-- context menu to manually fix rotation
+function onLoad()
+  self.addContextMenuItem("rotate preview", rotatePreview)
+  self.addContextMenuItem("rotate card+preview", function()
+    self.setRotationSmooth(self.getRotation() + Vector(0, 180, 0))
+    rotatePreview()
+  end)
+end
+
+-- rotates the alt_view_angle
+function rotatePreview()
+  local angle = self.alt_view_angle
+  if angle.y == 0 then
```

Comment:
Probably want a small range here in case the rotation gets imprecise on a save

Reply:
Only coordinates are rounded onSave, this value can't have a value besides 0 (or 180 if changed via this script)